cmd/fmt_test.go

```diff
@@ -153,3 +153,62 @@ p {
 		}
 	})
 }
+
+func TestFmtFailFileChanges(t *testing.T) {
+	params := fmtCommandParams{
+		fail: true,
+	}
+	var stdout bytes.Buffer
+
+	policyContent := `package test
+
+p {
+	a == 1
+	true
+	1 + 3
+}
+`
+
+	files := map[string]string{
+		"policy.rego": policyContent,
+	}
+
+	test.WithTempFS(files, func(path string) {
+		policyFile := filepath.Join(path, "policy.rego")
+		info, err := os.Stat(policyFile)
+		err = formatFile(&params, &stdout, policyFile, info, err)
```

Comment:
Looks like we don't care for `stdout`. We could pass [`io.Discard`](https://golang.org/pkg/io/#Writer) instead, I suppose.

Reply:
updated